src/keeper.js

```diff
@@ -99,6 +107,7 @@ export default class keeper {
       if(Object.prototype.hasOwnProperty.call(Config.vars.collateral,name)) {
         const collateral = Config.vars.collateral[name];
         this._clipperInit(collateral).then(pair => {
+          // add the pair to the array of clippers
```

Comment:
Can we find out what is the pair variable actually defined to? I'm personally curious :))

Reply:
Alright!